### PR TITLE
Add missing class

### PR DESCRIPTION
**Pull request by AinaSG**
Those 2 sponsors had the logo partially hidden and the wrong hover over animation.

### CODE REVIEW

File: themes/godotengine/layouts/home.htm [hunk 1]
```diff
@@ -306,10 +306,10 @@ <h2 class="text-center">Sponsored by</h2>
       <a class="sponsor card base-padding" href="https://www.gamblify.com/" target="_blank">
         <img src="{{ 'assets/home/sponsors/gamblify.png' | theme }}" alt="Gamblify">
       </a>
-      <a class="sponsor card" href="https://www.moonwards.com/" target="_blank">
+      <a class="sponsor card  base-padding" href="https://www.moonwards.com/" target="_blank">
```
Comment (Calinou) [inline themes/godotengine/layouts/home.htm]: I know I'm nitpicking, but there should only be one space to separate each CSS class :slightly_smiling_face:

File: themes/godotengine/layouts/home.htm [hunk 2]
```diff
@@ -306,10 +306,10 @@ <h2 class="text-center">Sponsored by</h2>
...
         <img src="{{ 'assets/home/sponsors/moonwards.png' | theme }}" alt="Moonwards">
       </a>
-      <a class="sponsor card" href="https://www.imagecampus.edu.ar/" target="_blank">
+      <a class="sponsor card  base-padding" href="https://www.imagecampus.edu.ar/" target="_blank">
```
Comment (Calinou) [inline themes/godotengine/layouts/home.htm]: Same as above.

**Review by Calinou — CHANGES_REQUESTED** [review]

File: themes/godotengine/layouts/home.htm (hunk 2 above)
Comment (AinaSG) [reply themes/godotengine/layouts/home.htm]: Upsies

**Comment by Calinou** [comment]
It seems the current [Moonwards logo](https://godotengine.org/themes/godotengine/assets/home/sponsors/moonwards.png) is cropped by design (to make its height more reasonable), so I'll close this.

Also, [Image Campus is no longer sponsoring Godot](https://github.com/godotengine/godot-website/commit/2c3bea7c894c15804297f2e8a8ee57d7fbae8775).

**Comment by AinaSG** [comment]
Then maybe it would be better crop the image? because the class with the animation and the margins is in all the other sponsors, right now it's not visible but if we add another sponsor without this one having the class, the animations break bad time.

**Pull request closed without merging**